snakes_cafe.py

```diff
@@ -0,0 +1,54 @@
+menus = {'Appetizers': {'Wings': 0, 'Cookies': 0, 'Spring Rolls': 0},
+         'Entrees': {'Salmon': 0, 'Steak': 0, 'Meat Tornado': 0,
+                     'A Literal Garden': 0},
+         'Desserts': {'Ice Cream': 0, 'Cake': 0, 'Pie': 0},
+         'Drinks': {'Coffee': 0, 'Tea': 0, 'Blood of the Innocent': 0}}
+
+
+def menu_welcome():
+    print('*' * 40)
+    print(('*' * 2) + '     Welcome to the Snakes Cafe!    ' + ('*' * 2))
+    print(('*' * 2) + ' ' * 36 + ('*' * 2))
+    print(('*' * 2) + '  To quit at any time, type "quit"  ' + ('*' * 2))
+    print('*' * 40)
```

Comment:
Just so you know, you can use triple quotes to do multiline strings, which would allow you to do a single `print()` statement.
```python
print('''
**********************************
**   Welcome to the Snakes Cafe!   **
**********************************
''')
```